doc/userguide/upgrade.rst

```diff
@@ -140,6 +140,10 @@ Logging changes
   live transaction, to reduce the chances of logging unrelated data.** This may
   lead to what looks like a regression in behavior, but it is a considered choice.
 
+Other Changes
+~~~~~~~~~~~~~
+- libhtp has been replaced with a rust version. This means libhtp is no longer built and linked as a shared library, and the libhtp dependency is now built directly into suricata.
+
```

Comment:
nit, probably, but isn't this a major change?
(although there are so many listed there that it may be more visible here xD )

Reply:
From the user perspective, this should not change anything :-p
(It simplifies stuff for people who compile Suricata)